util/status.cc

```diff
@@ -15,10 +15,11 @@
 namespace rocksdb {
 
 const char* Status::CopyState(const char* state) {
-  const size_t cch = 
+  const size_t cch =
       std::strlen(state) + 1; // +1 for the null terminator
   char* const result =
-      new char[cch]; 
+      new char[cch];
+  result[cch - 1] = '\0';
```

Comment:
oops, sorry I have trouble reading the new internal phabricator and thought it was telling me yesterday that all the tests passed..